src/TipBox.vue

```diff
@@ -1,24 +1,51 @@
 <template>
-    <div class="alert container" :class="[boxStyle, addClass, lightStyle]" :style="customStyle">
-        <div v-if="!isDefault" class="icon-wrapper">
-            <span v-html="iconType"></span>
+    <div>
+        <div v-if="isSeamless" class="alert admonitionblock">
+            <table>
+                <tbody>
+                <tr>
+                    <td v-if="!isDefault" class="icon">
```

Comment:
These changes seem overkill. Can't we achieve this by overriding styles?

Reply:
I referred to how AsciiDoc implements their boxes. Probably it's easier to maintain if we write two styles (i.e. seamless and normal) separately?

Reply:
How is it easier to maintain unnecessary copies and modifications?

Reply:
Thanks for your criticism. I re-write my implementation. Can you take a look?